### ISSUE TITLE
Score is mysterious

### ISSUE THREAD

**Issue by mkoniecz** [geometalab/OSMTagFinder#2]
Score included in search results is deeply mysterious and therefore not useful. It is unknown what it is measuring and whatever given value is low/high.

**Comment by s-gwerder** [comment]
Indeed, the score has not much use anymore as we sort the results by different means now. I'm thinking about removing it from the view entirely. Can be brought back when there's a sorting option perhabs.
The score indicates how well the search terms matched the result in respect to other results and a lot of parameters influence it, like where in the tag the term matched or if the term occured in the decription etc. (Okapi BM 25)
